delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java

```diff
@@ -49,7 +47,7 @@
   private static final String TARGET = "target";
   private static final String GENERATION = "generation";
   private static final String OFFSET_BASE_PATH = "offset.base.path";
-  private static final String SOURCE_TABLES = "source.tables";
+  private static final String READER_DEFINITION = "source.tables";
```

Comment:
ditto, should we change the value also to "reader.definition" for consistency?

Reply:
oops, yes will do that